fixtures/article.md

```diff
@@ -93,3 +93,5 @@ function $initHighlight(block, cls) {
 
 export  $initHighlight;
 ```
+
+:notice[This **content** is a generic directive!]{color=blue}
```

Comment:
Is this just an example?  This PR adds support for any directive and any attributes?

Reply:
yeah, this is just an example